evaluate_versions.py

```diff
@@ -189,18 +189,43 @@ def create_general_diff(workbook: xlsxwriter.Workbook,
     general_diff.write(general_diff_row, 3, first_dyn_inst_count - second_dyn_inst_count, dyn_inst_format)
 
 
+def delete_needless_collects(collects):
+    collects_exec_counts = {}
+
+    for collect in collects:
+        if "x264_r" in collect:
+
+            with open(collect, "r") as block_fp:
+                exec_count = int(((block_fp.readlines()[-2]).split(":"))[1])
+                collects_exec_counts[collect] = exec_count
+
+    collects = [item for item in collects if "x264_r" not in item]
+    filtered_collects = sorted(collects_exec_counts.items(), key=lambda x: x[1], reverse=True)
+
+    if len(filtered_collects) >= 3:
+        collects.append(filtered_collects[0][0])
+        collects.append(filtered_collects[1][0])
+        collects.append(filtered_collects[2][0])
+
+    return collects
```

Comment:
1. Rename argument and return variable names
2. If I am not mistaken fprate bench also has side collects that need to be removed.